From 13a8894fe2faa45b04033d7122a8fe7939ce6aa2 Mon Sep 17 00:00:00 2001
From: Gavin Smith <gavinsmith0123@gmail.com>
Date: Thu, 16 Sep 2021 19:06:28 +0100
Subject: [PATCH] * tp/Texinfo/XS/parsetexi/end_line.c
 (parse_line_command_args) <@headings>: Add missing option value 'single'. 
 Report from Sergei Trofimovich <slyich@gmail.com>.

---
 ChangeLog                          | 6 ++++++
 tp/Texinfo/XS/parsetexi/end_line.c | 4 ++--
 2 files changed, 8 insertions(+), 2 deletions(-)

diff --git a/tp/Texinfo/XS/parsetexi/end_line.c b/tp/Texinfo/XS/parsetexi/end_line.c
index a5bd0ce52a..c973cc1230 100644
--- a/tp/Texinfo/XS/parsetexi/end_line.c
+++ b/tp/Texinfo/XS/parsetexi/end_line.c
@@ -778,8 +778,8 @@ kdbinputstyle_invalid:
     case CM_headings:
       {
         if (!strcmp (line, "off") || !strcmp (line, "on")
-            || !strcmp (line, "double") || !strcmp (line, "singleafter")
-            || !strcmp (line, "doubleafter"))
+            || !strcmp (line, "single") || !strcmp (line, "double")
+            || !strcmp (line, "singleafter") || !strcmp (line, "doubleafter"))
           {
             ADD_ARG(line);
           }
-- 
2.38.1

